### ISSUE TITLE
Ubuntu 22.04 upgrade from 3.6 to 3.7 not working

### ISSUE THREAD

**Issue by acgafni** [KRTirtho/spotube#1556]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

running on the terminal i get these error messages (and it will not start working):
(spotube:454678): GLib-GObject-WARNING **: 16:19:50.084: invalid (NULL) pointer instance

(spotube:454678): GLib-GObject-CRITICAL **: 16:19:50.084: g_signal_connect_data: assertion 'G_TYPE_CHECK_INSTANCE (instance)' failed

** (spotube:454678): CRITICAL **: 16:19:50.084: FlView *fl_plugin_registrar_get_view(FlPluginRegistrar *): assertion 'FL_IS_PLUGIN_REGISTRAR(self)' failed

(spotube:454678): Gtk-CRITICAL **: 16:19:50.085: gtk_container_forall: assertion 'GTK_IS_CONTAINER (container)' failed

** (spotube:454678): CRITICAL **: 16:19:50.085: FlBinaryMessenger *fl_plugin_registrar_get_messenger(FlPluginRegistrar *): assertion 'FL_IS_PLUGIN_REGISTRAR(self)' failed

** (spotube:454678): CRITICAL **: 16:19:50.085: FlMethodChannel *fl_method_channel_new(FlBinaryMessenger *, const gchar *, FlMethodCodec *): assertion 'FL_IS_BINARY_MESSENGER(messenger)' failed

** (spotube:454678): CRITICAL **: 16:19:50.085: void fl_method_channel_set_method_call_handler(FlMethodChannel *, FlMethodChannelMethodCallHandler, gpointer, GDestroyNotify): assertion 'FL_IS_METHOD_CHANNEL(self)' failed

** (spotube:454678): CRITICAL **: 16:19:50.085: FlBinaryMessenger *fl_engine_get_binary_messenger(FlEngine *): assertion 'FL_IS_ENGINE(self)' failed

** (spotube:454678): CRITICAL **: 16:19:50.085: FlTextureRegistrar *fl_engine_get_texture_registrar(FlEngine *): assertion 'FL_IS_ENGINE(self)' failed

** (spotube:454678): CRITICAL **: 16:19:50.085: FlPluginRegistrar *fl_plugin_registrar_new(FlView *, FlBinaryMessenger *, FlTextureRegistrar *): assertion 'FL_IS_BINARY_MESSENGER(messenger)' failed

(spotube:454678): GLib-GObject-CRITICAL **: 16:19:50.085: g_object_ref: assertion 'G_IS_OBJECT (object)' failed

** (spotube:454678): CRITICAL **: 16:19:50.085: FlBinaryMessenger *fl_plugin_registrar_get_messenger(FlPluginRegistrar *): assertion 'FL_IS_PLUGIN_REGISTRAR(self)' failed

** (spotube:454678): CRITICAL **: 16:19:50.085: FlMethodChannel *fl_method_channel_new(FlBinaryMessenger *, const gchar *, FlMethodCodec *): assertion 'FL_IS_BINARY_MESSENGER(messenger)' failed

** (spotube:454678): CRITICAL **: 16:19:50.085: void fl_method_channel_set_method_call_handler(FlMethodChannel *, FlMethodChannelMethodCallHandler, gpointer, GDestroyNotify): assertion 'FL_IS_METHOD_CHANNEL(self)' failed

** (spotube:454678): CRITICAL **: 16:19:50.085: void fl_method_channel_invoke_method(FlMethodChannel *, const gchar *, FlValue *, GCancellable *, GAsyncReadyCallback, gpointer): assertion 'FL_IS_METHOD_CHANNEL(self)' failed

### Expected Behavior

play music...

### Steps to reproduce

can not get the program to run - installed from deb file
the 

### Operating System

Ubuntu 22.04

### Spotube version

3.7

### Installation source

Website (spotube.netlify.app) or (spotube.krtirtho.dev)

### Additional information

no

### Self grab

- [ ] I'm ready to work on this issue!

**Comment by OpenCode** [comment]
Same here

**Comment by guanciottaman** [comment]
Me too

**Comment by KRTirtho** [comment]
Duplicate of #1555

**Comment by acgafni** [comment]
yes - it is a duplicate of https://github.com/KRTirtho/spotube/issues/1555, sorry
